config/initializers/notifications.rb

```diff
@@ -1,4 +1,5 @@
 NOTIFY_KEY = ENV["NOTIFY_KEY"]
+NOTIFY_APPLICATION_SUBMITTED_CONFIRMATION_TEMPLATE = "0a75c46b-923f-4ee0-807c-99fd289a881a".freeze
```

Comment:
Maybe `_JOBSEEKER_` would be useful, or maybe it's too cumbersome?

Reply:
It's a fair cop.